app/services/submission_service.rb

```diff
@@ -101,6 +103,12 @@ def update_submission(
       submission.save!
     end
 
+    def notify_user_with_braze(submission_id, response)
+      if response
+        SwaMyCollectionEmailWorker.perform_in(1.day.from_now, submission_id)
```

Comment:
I decided to use a worker instead of the `delay_until` just to have a better overview of failures and such within sidekiq.

Reply:
That's a great idea! Do we have the option to re-send any failed attempts with a worker?